src/actions/_rsaa.js

```diff
@@ -1,8 +1,61 @@
+import _ from 'lodash';
+
+/**
+ * Check if headers object supports entities() method.
+ * @param {Headers} headers
+ */
+function isHeadersEntriesSupported(headers) {
+  return (_.isFunction(headers.entries));
+}
+
+/**
+ * Return existing or create new Headers object instance
+ * @param {Headers|Object} headers
+ */
+function getHeadersInstance(headers) {
+  if (headers instanceof Headers) {
+    return headers;
+  }
+  return new Headers({ ...headers });
+}
+
+/**
+ * Extract response headers into a plan object
+ * @param {Object} response
+ */
+export function extractResponseHeaders(response) {
+  if (!response || !response.headers) {
+    return {};
+  }
+
+  const headersInstance = getHeadersInstance(response.headers);
+  const headers = {};
+  let currentHeader = null;
+
+  if (isHeadersEntriesSupported(headersInstance)) {
+    // Headers extract implementation for React Native
+    const iterator = headersInstance.entries();
+    for (currentHeader = iterator.next(); !currentHeader.done; currentHeader = iterator.next()) {
```

Comment:
Browser support? As you said, this is redux lib and it can be run on any env and any browser. for...of has limited browser support when it comes to IE.